SPECS-EXTENDED/freeradius/freeradius.spec

```
@@ -2,8 +2,8 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Summary: High-performance and highly configurable free RADIUS server
 Name: freeradius
-Version: 3.0.21
-Release: 9%{?dist}
+Version: 3.2.3
+Release: 1%{?dist}
 License: GPLv2+ and LGPLv2+
 URL: http://www.freeradius.org/
```

Let's use `https` if it has a TLS enabled website.

```suggestion
URL: https://www.freeradius.org/
```